ocs_ci/ocs/scale_lib.py

```diff
@@ -978,3 +978,17 @@ def check_all_pvc_reached_bound_state_in_kube_job(
             logging.info("All PVCs in Bound state")
             break
     return pvc_bound_list
+
+
+def get_max_pvc_count():
+    """
+    Return the maximum number of pvcs to test for.
+    This value is 500 times the number of worker nodes.
+    """
+    ocp_info = OCP().exec_oc_cmd("get nodes -o json")
+    count = 0
+    for node_chk in ocp_info["items"]:
```

Comment:
We should be looking for worker nodes running OCS, not just the worker nodes.

Reply:
How do I figure out if a worker node is running OCS?   Wouldn't this make the number of nodes I find less than what is found here?  (Number of worker nodes running  OCS <=  Number of worker nodes)

Reply:
I have implemented the changes in the latest version.